rooms.py

```diff
@@ -94,139 +121,130 @@ def room_allocation(self, args):
             room_type = "LIVING SPACE"
             occupants = living_space.living_space_occupancy(living[0])
         else:
-            print("No room exists in amity with that name. please try again")
-            return
+            return "No room exists in amity with that name. please try again"
```

Comment:
**General - Nitpick:** This should be grammatical. i.e. `"No room exists in Amity with that name. Please try again."`